girder/api/rest.py

```diff
@@ -18,6 +18,7 @@
 ###############################################################################
 
 import cherrypy
+import cherrypy.lib
```

Comment:
Oops, I'm totally wrong. I was using it with a different import style during my development, and that wasn't working. The extra explicit import of "cherrypy.lib" is not necessary.

I'll fix this.